### DISCUSSION TITLE
Intentando hacer deploy de backend

### DISCUSSION THREAD

**Discussion by Asilvazavala** [vercel/community#3452] [Help]
### Summary

Hola, soy nuevo en esto, creé una base de datos con PostgreSQL en vercel, luego creé un proyecto backend y le añadí las variables de entorno, me tira este error al hacer el deploy de la aplicación backend:

This Serverless Function has crashed.
Your connection is working correctly.
Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED
ID: sfo1::jnqk6-1690853197143-14aa877c7833

Añadí las variables de entorno de la base de datos creada en vercel al .env del proyecto, lo probé en el localhost y si me funciona, pero el deploy tira ese error, ¿Alguien me podría decir como puedo resolverlo?

### Example

https://pokemon-world-back.vercel.app/

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/103531207/877de6e2-ef1e-43eb-b947-c6a78f5a4958)

**Comment by amyegan** [comment]
Hola @Asilvazavala. Lo siento, pero estoy aprendiendo español todavía. Sólo hablo inglés.

The [runtime logs](https://vercel.com/docs/concepts/observability/runtime-logs#) may have more information about the error. I often wrap function code in a [try...catch statment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) so I can log additional error information. This works well with runtime logs. 

If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here may be able to recognize what is going wrong.
